handbook/engineering/distribution/onboarding.md

```diff
@@ -0,0 +1,81 @@
+# Distribution Engineer Oboarding
+
+Welcome to the distribution team! This document will guide you through distribution-specific onboarding tasks to make your start here at Sourcegraph as smooth as it can be.
+
+## Distribution Onboarding Checklist
+
+### Week 1
+
+#### Goals:
+- Clear knowledge of what the company’s, and Distributions’s mission, vision, values and culture are.
+- Know the expected outcome of each Distribution onboarding milestone.
+- Become familiar with Distribution [product](product.md),[tech stack](tech_stack.md) and [processes](recurring_processes.md).
+- Go through [General Onboarding](../../people-ops/onboarding/general_onboarding.md)
+- You have a sense of what tools you will be using and for what
+
+
+#### Tasks:
+- Meet your onboarding buddy
+- Add shared [Distribution calendar](https://calendar.google.com/calendar/u/0?cid=Y19qZHU3NTJ2anFpZ2NnMWVmYmYxZXIzY291Z0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to your calendar
+- Attend weekly sync [meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=OXFyZXRsc21hdDh1MHNkNzQ2aGhqMzg3NmxfMjAyMTA1MjRUMTczMDAwWiBkYXZlQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=dave%40sourcegraph.com&scp=ALL)
+- Deploy your own SG instance using one of the 3 deployment methods
+- Familiarise yourself with the distribution [internal infrastructure](internal_infrastructure.md) and [ownership areas](ownership_areas.md)
+- Open and merge first GitHub pull request by adding yourself to [team page](https://about.sourcegraph.com/company/team) in Handbook
+- Read our [Distribution handbook pages](index.md)
+- Join the distribution Slack channels
+  - #distribution
+  - #distribution-team
+  - #dev-ops
+  - #dev-chat
+  - #dev-accounce
+  - #alerts-distribution
+
+### Week 2
+
+#### Goals:
+- Develop high-level understanding of product.
+- You are participating in our team rituals.
+- Get to know your team.
+- Close at least one issue.
```

Comment:
```suggestion
- Work towards closing one issue.
```
I don't think we want to make someone feel like they are falling behind if they don't close an issue by here (took me about 3 weeks to onboard and finish up the version warnings issues)